c#/Mandoline.Api.Examples/Tests/GlobalSuppressions.cs

```diff
@@ -0,0 +1,33 @@
+﻿// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1600:Elements should be documented", Justification = "<Pending>", Scope = "member", Target = "~F:Tests.TestOutput.ReturnValueDate")]
```

Comment:
As before we could use a single project level suppression rule here which would remove all of the SA1600 lines this file contains making it easier to read. Of course this depends on if we are happy to ignore all documentation.